apps/web/composables/useProductAttributes/__tests__/useProductAttributes.spec.ts

```diff
@@ -0,0 +1,56 @@
+import ProductFixture from "./../../../utils/__tests__/__fixtures__/Product";
+
+describe('useProductAttributes', () => {
+    it('initializes state correctly', () => {
+        const { attributes, attributeValues, combinations, itemId, variationId } = useProductAttributes();
+
+        expect(attributes.value).toEqual([]);
+        expect(attributeValues.value).toEqual({});
+        expect(combinations.value).toEqual([]);
+        expect(itemId.value).toBe(0);
+        expect(variationId.value).toBe(0);
+    });
+
+    it('updateValue updates attribute values', () => {
+        const { setAttribute, updateValue, attributeValues } = useProductAttributes();
+        setAttribute(ProductFixture, true);
+        updateValue(1, 1);
+
+        expect(attributeValues.value).toEqual({ 1: 1, });
+    });
+
+
+    it('updateValue should return empty object if value is not found', () => {
```

Comment:
```suggestion
    it('should should return empty object if value is not found during update', () => {
```

Reply:
**question**: Just to be sure, does this only refer to the initial state or is this always true? In other words, what should happen if `attributesValues.value` is already set and you try to update with an invalid value? Does it get unset or keep the previous value?

Reply:
Each time setAttribute is called, state.value.attributeValues = {}. The second parameter of the function, if true, will set the attribute values of the product send in first parameter. So every time the function is called, new attribute values pop will come